### DISCUSSION TITLE
Invoice download PDF Error

### DISCUSSION THREAD

**Discussion by sittiphong** [boxbilling/boxbilling#1086] [Q&A] [answered]
Hi,
I see bug in billing system, After click Download PDF

![invoice-1](https://user-images.githubusercontent.com/34829333/138025019-2738442e-fbff-4109-9442-ce63630d6559.png)

Error code
![invoice-2](https://user-images.githubusercontent.com/34829333/138025022-c1ea80fc-e3b8-475b-87b8-6aff0ee811cc.png)

Do you have any solution ?

Thanks.

**Comment by BelleNottelling** [comment]
This PR has all of the changes that are required to fix PDF generation:
https://github.com/boxbilling/boxbilling/pull/927

**Comment by BelleNottelling** [comment, marked as answer]
Fixed in last release (plus other problems)
https://github.com/boxbilling/boxbilling/releases
